public/scripts/raneto.js

```diff
@@ -18,8 +18,7 @@
       $(".content table").addClass("table");
 
       // FitVids
-      $(".content").fitVids();
-
+      fitVids();
```

Comment:
Yes, please change it to the `fitvids(".content");`
You can definitely leave a comment there for people that may need multiples.

Also, some more (picky) requests coming.
Makes the diff history easier, so sorry to be a pain.
Other than that, ready to merge.